packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -150,6 +151,7 @@ export class AtomicBreadcrumbManager {
       <li part="breadcrumb-value" class="pr-3">
         <button
           part="breadcrumb-button"
+          aria-label={`Remove inclusion filter on ${breadcrumbValue.value.start} to ${breadcrumbValue.value.end}`}
```

Comment:
Will a sceen-reader correctly read a date range? Or do we need to format the `start` and `end` in a certain way?

Reply:
hmmm, it reads character/number on at a time. 
e.g. 2008/01/01@00:00:00, it reads : 2008 slash O 1 slash 0 1 at 0 0 colon 0 0 colon 0 0

Reply:
This is usually a setting in the screen reader to read numbers properly.

Reply:
I think we need to format the dates. I'll use `Date` to format it to remove the unnecessary 0s and have a better formatted time